pkgs/top-level/all-packages.nix

```diff
@@ -26344,6 +26344,7 @@ with pkgs;
   prometheus-collectd-exporter = callPackage ../servers/monitoring/prometheus/collectd-exporter.nix { };
   prometheus-consul-exporter = callPackage ../servers/monitoring/prometheus/consul-exporter.nix { };
   prometheus-dcgm-exporter = callPackage ../servers/monitoring/prometheus/dcgm-exporter { };
+  prometheus-deluge-exporter = callPackage ../servers/monitoring/prometheus/deluge-exporter.nix { };
```

Comment:
About the failing pkgs/by-name check you mentioned earlier, I'd like you to have a look at https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md. Nixpkgs is moving towards using these as the default package location, and new packages are not allowed to go into `top-level/all-packages.nix` (with some exceptions). The only action required is to move `pkgs/servers/monitoring/prometheus/deluge-exporter.nix` to `pkgs/by-name/de/deluge-exporter/package.nix` and delete the entry from `all-packages.nix`. The check should run fine after this.

Reply:
Sure that works. I did that for other packages. I wasn’t sure if here was one of those exceptions or not since so many exporters are here too.